src/mgr/MgrPyModule.cc

```diff
@@ -171,30 +171,40 @@ int MgrPyModule::load()
   }
 
   // Find the class
-  // TODO: let them call it what they want instead of just 'Module'
-  auto pClass = PyObject_GetAttrString(pModule, (const char*)"Module");
-  Py_DECREF(pModule);
-  if (pClass == nullptr) {
+  auto  mgr_module_type =  PyImport_ImportModule("mgr_module.MgrModule");
+  if (mgr_module_type == nullptr) {
     derr << "Class not found in module '" << module_name << "'" << dendl;
```

Comment:
change the error message to something like

> Unable to import mgr_module from MgrModule

as we are not importing the `Module` from from `$module_name` here.